wwwroot/assets/scripts/api.js

```diff
@@ -0,0 +1,65 @@
+import * as g from './global';
```

Comment:
improve naming.

Reply:
Is it really bad practice?
It's used everywhere, so I figured a shortcut like the $ for document.getElementBySomething would be ok.

You think all the g.xxx should become global.xxx for example?